open-api-spec.yaml

```diff
@@ -84,6 +84,24 @@ paths:
 
         The content should be slow changing and cacheable for long periods. Servers SHOULD use cache control headers.
       security: []
+  /jwks.json:
```

Comment:
https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-key-sets
>Auth0 exposes a JWKS endpoint for each tenant, which is found at `https://YOUR_DOMAIN/.well-known/jwks.json`

That makes wonder if a payment pointer such as `https://alice.me/.well-known/pay` should have `https://alice.me/.well-known/jwk.json`

Reply:
I'm not sure if I understand the question.

> That makes wonder if a payment pointer such as https://alice.me/.well-known/pay should have https://alice.me/.well-known/jwk.json

This proposal would resolve exactly like this. i.e. If you had a payment pointer `$alice.me` then your payment pointer URL would be `https://alice.me/.well-known/pay` and your keys would be at `https://alice.me/.well-known/jwk.json`

Reply:
As is, the payment pointer would be:
`https://alice.me/.well-known/pay`
and the keys would be at:
`https://alice.me/.well-known/pay/jwk.json`

I'm wondering if it's worth following the auth0 convention and putting the keys at:
`https://alice.me/.well-known/jwk.json`

(which would only affect `/.well-known/pay` path prefix payment pointers)

Reply:
>(which would only affect /.well-known/pay path prefix payment pointers)

An example of a non `/.well-known/pay` payment pointer:

`$wallet.example/alice`
`https://wallet.example/alice`

in which case it seems like the keys would have to be at:
`https://wallet.example/alice/jwk.json`